src/renderer/views/wallet/Interact/CustomView.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react'
+
+import * as RD from '@devexperts/remote-data-ts'
+import { BaseAmount } from '@xchainjs/xchain-util'
+import * as FP from 'fp-ts/function'
+import * as O from 'fp-ts/Option'
+import { useIntl } from 'react-intl'
+import * as Rx from 'rxjs'
+
+import { Custom } from '../../../components/interact/forms/Custom'
+import { Button } from '../../../components/uielements/button'
+import { useChainContext } from '../../../contexts/ChainContext'
+import { useThorchainContext } from '../../../contexts/ThorchainContext'
+import { INITIAL_INTERACT_STATE } from '../../../services/thorchain/const'
+import { InteractState } from '../../../services/thorchain/types'
+import * as Styled from './InteractView.styles'
+
+type Props = {
+  goToTransaction: (txHash: string) => void
+}
+
+export const CustomView: React.FC<Props> = ({ goToTransaction }) => {
+  const [interactState, setInteractState] = useState<InteractState>(INITIAL_INTERACT_STATE)
+  const { interactService$ } = useThorchainContext()
+  const { txStatus$ } = useChainContext()
+  const intl = useIntl()
+
+  const possibleSubRef = useRef<O.Option<Rx.Subscription>>(O.none)
```

Comment:
[nitpick] `possibleSubRef` -> `oSubRef` or just `subRef`. We don't need to name `possibleXYZ` `Option` values.